.gitignore

```diff
@@ -12,9 +12,12 @@ typo3_src
 web/index.php
 web/typo3
 
-web/typo3conf/ext
+web/typo3conf/
+!web/typo3conf/ext
```

Comment:
Shouldn't `web/typo3conf/ext` be gitignored so that the extension code won't end up in the repo, but OTOH be want to keep LocalConfiguration and PackageStates?

So perhaps just add a line to ignore the deprecation log?

Reply:
You're right. Usually the `typo3conf/` is ignored and the `LocalConfiguration.php` is excluded. 

It's not only the ` typo3conf/ext` directory which muste be ignored but also ` typo3conf/l10n`, ` typo3conf/deprecation*.log`,  ` typo3conf/ENABLE_INSTALL_TOOL` and maybe more.